### ISSUE TITLE
Drag and Drop interface doesn't work

### ISSUE THREAD

**Issue by TheEvilSkeleton** [sonnyp/Workbench#460]
To reproduce, Library -> Drag and Drop, then attempt to drag and drop. Pressing and holding won't do anything.

[Screencast from 2023-07-30 11-21-12.webm](https://github.com/sonnyp/Workbench/assets/50847364/8737f628-94ac-4c9f-8d95-fa1a3a387365)

**Comment by andyholmes** [comment]
Did you press the "Run" button before trying the demo? I believe there is an auto-run feature planned (or in the development branch), but for now they have to be run to get the code active.

**Comment by TheEvilSkeleton** [comment]
No, I didn't press the "Run" button. That made it work, thanks :)

Yeah, an auto-run feature would certainly help.

**Comment by sonnyp** [comment]
We already have auto-run support, for some reason it's not enabled in this demo https://github.com/sonnyp/Workbench/blob/main/src/Library/demos/Drag%20and%20Drop/main.json

@TheEvilSkeleton would you like to send a fix?

**Comment by TheEvilSkeleton** [comment]
Sure :)
